### PR TITLE
scroll top with dummy content example

### PR DESCRIPTION
**Pull request by AdityaVandan**
scroll-top (taken from codebase)

class component converted to functional component

### CODE REVIEW

File: src/utils/content/index.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+export const scrollTopDummyContent = () => (
```
Comment (ankitchouhan1020) [inline src/utils/content/index.tsx:3]:
```tsx
export const scrollTopDummyContent = () => (
  <div style={{ minHeight: 2000}}>Scroll to bottom</div>
)
```

File: stories/ScrollTop.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { action } from '@storybook/addon-actions';
+
+import { Props as ScrollTopProps } from '../src/components/atoms/ScrollTop/ScrollTop';
+import { ScrollTop } from '../src/components/atoms';
+import { scrollTopDummyContent } from '../src/utils/content';
+
+export default {
+  title: 'ScrollTop',
+  component: ScrollTop,
+  argTypes: {
```
Comment (ankitchouhan1020) [inline stories/ScrollTop.stories.tsx:12]: remove if not required.

File: stories/ScrollTop.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <>
+      {scrollTopDummyContent()}
+      <ScrollTop {...args} />
+    </>
+  );
+}
+
+export const Default = Template.bind({});
+Default.args = {
```
Comment (ankitchouhan1020) [inline stories/ScrollTop.stories.tsx:26]: remove if not required

File: src/components/atoms/ScrollTop/ScrollTop.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    </div>
+  )
+}
+
+const defaultProps: DefaultProps = {
+  style: {}
+}
+
+
+type RequiredProps = {
```
Comment (ankitchouhan1020) [inline src/components/atoms/ScrollTop/ScrollTop.tsx:68]: remove if not required, use only DefaultProps

**Review by ankitchouhan1020 — CHANGES_REQUESTED** [review]

File: src/utils/content/index.tsx (hunk 1 above)
Comment (AdityaVandan) [reply src/utils/content/index.tsx:3]: done

**Review by ankitchouhan1020 — APPROVED** [review]

**Review by dheerajsingh123 — APPROVED** [review]

**Pull request merged**